GoInfoGame/GoInfoGameTests/quests/BusStopLitTests.swift

```diff
@@ -41,13 +41,18 @@ final class BusStopLitTests: XCTestCase {
      */
     
     func testBusStopLitQuery() throws {
+        let eightYearsAgo = TestQuestUtils.olderThan(years: -8)
+        let sixteenYearsAgo = TestQuestUtils.olderThan(years: 8)
```

Comment:
why is this advancing another 8 years? where is this used?

Reply:
removed the unused variable